src/pipeline.cc

```diff
@@ -688,8 +688,14 @@ class PipelineWorker : public Napi::AsyncWorker {
         image = sharp::Gamma(image, baton->gammaOut);
       }
 
+      // Linear arg check
```

Comment:
Can this array length check move to occur earlier, in the JavaScript rather than C++? We should probably add a test for it too.